### ISSUE TITLE
starting LimaCCDs without the hardware

### ISSUE THREAD

**Issue by PhilLAL** [esrf-bliss/Lima#102]
Hi, 
I would like to know if there is a way to start LimaCCDs without the hardware accessible through the network.
It could allow to start it even if users did not have already the device.
Currently, a device can be disconnected from the network and LimaCCDs still runs, but it needs to be connected during startup.

Cheers.

**Comment by pufuleterau** [comment]
You can try to use SimulationCCD, I m not 100% sure will work.

**Comment by PhilLAL** [comment]
Thank you for this idea. Do you talk about Simulator LImA plugin ?
If yes, I was more interested into starting the real DS without the hardware being connected, and then connect the hardware. I will find another way.

**Comment by pufuleterau** [comment]
Yes I talk about Simulator Lima plugin.

**Comment by pufuleterau** [comment]
Ok I did the following tests, but first I need to tell you that I have already created my DS(LimaCCD) using a hardware:
1. I disconnect the camera and I was able to run the DS without problem using Simulator plugin.
2. I tryed without Simulator plugin (as I want to connect to the hardware), the DS started but I have some errors because I have some parameters that interogate the hardware , because no hardware was available I got some errors but the DS was ready.

In conclusion from my point of view if you create your DS with parameters that are not mandatory to be  received from hardware (actually you avoid to comunicate with the hardware and "start only the process"), you'r LimaCCD DS should work. I think 💃

**Comment by PhilLAL** [comment]
Yes, in fact my conclusions are the same, I forgot to precise that I tested this on a Basler, which use a property to define the IP:)

**Comment by pufuleterau** [comment]
I m glad we arive at the same conclusio. I hope will help you this.

Have a good day!
Cheers
